content/download/mirrors.adoc

```diff
@@ -7,15 +7,49 @@
 - infrastructure
 :author: rtyler
 ---
-## Releases
 
-Currently releases are mirrored to the locations documented on the link:https://get.jenkins.io/index.html?mirrorstats[mirror status page].
+Jenkins packages and plugins are served by the web service https://get.jenkins.io (or its legacy alias mirrors.jenkins.io).
 
-### Running a mirror
+The web service https://get.jenkins.io relies on a network of mirror servers provided by generous sponsors of the Jenkins project all over the world.
+When a user download a plugin or a file, the web service https://get.jenkins.io redirects (transparently) the user to the closest mirror.
+The web service https://get.jenkins.io is based on the Open Source mirror redirector https://github.com/etix/mirrorbits.
 
-More mirrors are certainly welcome.
-Jenkins is mirrored by the link:https://osuosl.org/services/hosting/details[Oregon State University Open Source Lab (OSUOSL)] and by servers hosted by donors in the United States, Europe, China, Singapore, and Japan.
-Worldwide mirrors reduce the load on individual servers, spread the bandwidth costs among multiple servers, and improve download performance for users.
+The list of mirrors (and their locations) is available on the following page: link:https://get.jenkins.io/index.html?mirrorstats[mirror status page].
 
-If you're interested in running a public mirror, open an issue in the link:https://github.com/jenkins-infra/helpdesk[Jenkins infra help desk] and we can discuss space and bandwidth requirements.
-Mirrors on the Indian subcontinent, in South America, Russia and in Africa would be great additions to the Jenkins mirrors.
+## How to Help the Jenkins Project by Running a Mirror?
```

Comment:
Overall feedback:

There's a lot of text here it would be good to either provide an issue form template or an example input so someone can copy paste and edit if required.

an issue form would be best

Reply:
I thought the same when opening the PR: this page has too many textual information and it might not be the best place for the "reference documentation for mirror maintainers".

WDYT if we move the "list" in a markdown in jenkins-infra/documentation as a dedicated page and link it there instead? 

I don't mind the issue form template, but it would be only cover the case of "adding a new mirror".

Reply:
Oh I see, the issue is not having "too much data" but more the rendering (e.g. hard to browse or read at first sight). Ack, then let me try something then. thanks!

Reply:
@timja done, I've implemented the list as subsections. How do it fare?